app/pages/onboarding/07-verify-key/verify-key.tsx

```diff
@@ -15,22 +18,46 @@ import {
 } from '../../../components/onboarding';
 
 export const VerifyKey: React.FC = () => {
+  const history = useHistory();
+  const mnemonic = useSelector(selectMnemonic);
+  const [inputMnemonic, setInputMnemonic] = useState('');
+  const [hasSubmitted, setHasSubmitted] = useState(false);
+
+  const mnemonicCorrect = mnemonic === inputMnemonic;
+
+  // https://github.com/blockstack/ux/issues/421
+  // See should really be a HTMLTextareaElement
+  const handleMnemonicInput = (e: React.FormEvent<HTMLInputElement>) => {
+    setInputMnemonic(e.currentTarget.value.trim());
```

Comment:
Do you think we should clear `hasSubmitted` here? Otherwise, I think the "Mnemonic is incorrect" will still display after you've started typing again. Maybe that's desired.

Reply:
Yeah, that's correct. This form validates immediately, after first submit, so you get instant feedback it's incorrect. I reckon this makes sense for this form (not all). Happy to change later if it doesn't work well.